homeassistant/components/climate/ecobee.py

```diff
@@ -269,6 +269,7 @@ def device_state_attributes(self):
             "fan": self.fan,
             "mode": self.mode,
             "operation": operation,
+            ATTR_OPERATION_LIST: self._operation_list,
```

Comment:
Yes ! But I don't remember why :/
I thought that adding the `operation_list` attribute shows all supported modes by the ecobee in the UI like the following screen.
To get some like this : 
![ecobee](https://cloud.githubusercontent.com/assets/47721/24068690/d7ec0fcc-0b6b-11e7-902a-e59ded55b265.jpg)

But I just removed this line and I still have the same choices ... So i guess this is useless...